### DISCUSSION TITLE
Disable profile banners option

### DISCUSSION THREAD

**Discussion by MinecraftHDKIll** [NamelessMC/Nameless#2135] [Ideas]
Make option to remove profile banners from profiles.

**Comment by Derkades** [comment]
Related to https://github.com/NamelessMC/Nameless/issues/1863

**Comment by MinecraftHDKIll** [comment]
Not related to that for sure. I'm talking about removing profile banners from profiles completely.
![image](https://user-images.githubusercontent.com/42026909/107115611-22889600-686e-11eb-8252-dfe7c672d582.png)
I want option to remove banner from it completely. Or if that is not possible add option to set default banner for everyone.
